test/vanilla/legacy/Expected/AcceptanceTests/MergePatchJson/mergepatchjson/aio/operations/_object_type_client_operations.py

```diff
@@ -63,7 +63,6 @@ async def patch_single(self, body: Any, **kwargs: Any) -> None:
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             error = self._deserialize.failsafe_deserialize("object", pipeline_response)
             raise HttpResponseError(response=response, model=error)
-
```

Comment:
do you know why there's different white space for mergepatchjson? Doesn't look like there were any code changes that would have led to it, maybe did you accidentally format it my opening the generated code?

Reply:
Be strange that this happen when I generate the test project in windows, and don't happen in Linux. So I committed the Linux result to undo the changes.

But I guess it's not related with this PR because this also happen in branch autorestv3 in my local machine. I will investigate further when have time.